### ISSUE TITLE
validator - minFiles

### ISSUE THREAD

**Issue by alexander-ronin** [yiisoft/yii2#15417]
Yii2 validator has `maxFiles ` property. But there is no `minFiles` in documentation.
I guess it's strange and  `minFiles` should be added too, which will checks for minimum quantity of uploaded files.
Thanks for attention.

**Comment by samdark** [comment]
What's the use case?

**Comment by alexander-ronin** [comment]
In form users must to upload at least 2 images.

**Comment by samdark** [comment]
OK. Not against adding it but since 2.0.14 will be the last feature-release and there are many things we want to get it, the only chance is if you'll prepare a pull request. Else it will be added into 2.1.

**Comment by alexander-ronin** [comment]
Got it, will wait 2.1.
Thanks )

**Comment by vladis84** [comment]
Hello. 
I will do it.

**Comment by vladis84** [comment]
I think it's right to specify the interval.
Example 
```php
['min' => 2, 'max' => 4];
['min' => 2, 'max' => 0]; // 0 - unlimited
```

**Comment by FeR-S** [comment]
Is this fixed in version 2.0.15.1?

**Comment by samdark** [comment]
2.0.14
